tests/suites/cli/clouds/mycreds.yaml

```diff
@@ -0,0 +1,7 @@
+credentials:
+  my-ec2:
+    mine:
+      auth-type: access-key
+      access-key: my-key
+      secret-key: my-secret
+
```

Comment:
remove empty line (?)

Reply:
doesn't make a difference either way, but cleaner to remove